about.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>About -  Capstone</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <div class="icon" id="hamburger">
+                <i class='bx bx-menu hamburger'></i>
+            </div>
+            <div class="logo">
+                <a href="index.html">Logo</a>
+            </div>
+            <ul class="nav-list">
+                <li class="nav-item">
+                    <div class="icon">
+                        <i class='bx bx-x close' id="close-icon"></i>
+                    </div>
+                </li>
+                <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+                <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+            </ul>
+        </nav>
+        <div class="hero-container">
+            <div class="hero-about-content">
+                <h1 class="hero-about-content__heading">
+                    CREATIVE COMMONS GLOBAL SUMMIT 2021
+                </h1>
```

Comment:
Apply the given design here  as shown below
![right2](https://user-images.githubusercontent.com/58376020/123692990-c5f73900-d857-11eb-9256-12a30714866b.png)

Reply:
This change is yet to be made. Kindly check below for more description about it